packages/common/src/hooks/purchaseContent/usePurchaseContentFormConfiguration.ts

```diff
@@ -60,6 +160,15 @@ export const usePurchaseContentFormConfiguration = ({
   const isUnlocking = !error && isContentPurchaseInProgress(stage)
   const { data: balanceBN } = useUSDCBalance()
   const balance = USDC(balanceBN ?? new BN(0)).value
+  const [guestEmail, setGuestEmail] = useLocalStorage('guest-email', '')
```

Comment:
why do we want this in `localStorage`? is it for users who want to make two purchases in the same guest session?

Reply:
also if we keep it in local storage, the key should probably be a constant

Reply:
yup exactly for subsequent purchases but i could leave it out for now, ill use that constant key